### DISCUSSION TITLE
Restrict issue opening to legit GitHub users

### DISCUSSION THREAD

**Discussion by sambacha** [MetaMask/eth-phishing-detect#11726] [Ideas]
There are way too many issues to be handled by a team, and that's because the majority of these issues are fake accounts requesting to unblock obvious phishing sites.

Solution: https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository

or alt.  create a github action similar to https://github.com/marketplace/actions/faceless-issues but without the comment disclosing about the avatar check and an additional check to see how many repos, commits or whatever other heuristic is relevant 

let me know if I can help

**Comment by sambacha** [comment]
@409H  @devenblake pls

**Comment by filosofo33** [comment]
what about domain speculators?
